checks/run-validation.sh

```diff
@@ -1,31 +1,19 @@
 #!/bin/bash
 # Testscript running validation tests according to GitHub Actions but without comparison to master
 #
-# Copyright (c) 2023 Simon Homes
+# Copyright (c) 2023      Simon Homes
+# Copyright (c) 2024      Christoph Niethammer <niethammer@hlrs.de>
 #
 
 repoPath="$PWD/.."  # Path to root directory
 
 export OMP_NUM_THREADS=2
 
+inputlist="${repoPath}/examples/example-list.txt"
 logfileName="${repoPath}/examples/run-validation.log"
 rm -f ${logfileName}
 
-IFS=$'\n'
-for i in $(cat "${repoPath}/examples/example-list.txt" )
-do
-	# skip if comment or empty line
-	if [[ $i == \#* || -z "$i" ]]
-	then
-		continue
-	fi
-	cd ${repoPath}/examples/$(dirname $i)
+cd ${repoPath}/examples
+./run-examples.sh -v --inputlist "${inputlist}" --logfile "${logfileName}"
```

Comment:
You could also use the options `--{mardyn,mpirun}_{exe,args}` here. Or we could just get rid of the entire file since its use case is already covered by the default settings of `run-examples.sh`, isn't it?

Reply:
Good point. I could not find any use case inside the codebase/workflows so I would get rid of the validation.sh script then.